NewForce-at-Mountwest/bangazon-site-take-2-fluffy
# Cancelling an order

# Description
Users now have the ability to cancel an order before it is completed.


## Related Ticket(s)
9--https://trello.com/c/4He4INx3/9-9-users-can-cancel-an-order


## Steps to Test
--Save, add and commit on your branch before checking out to sw-cancel-order
--run the app and create an order using the "add to cart" button on any product
--Add a few different products, then go to "View Cart"
--You should see a link to 'cancel order' below the order title
--Clicking on this will bring you to a confirmation page which shows the details of the order and asks if you are sure you want to cancel.
--Clicking delete will delete the join tables for the line items in the order (order product) and then delete the order
--You will redirect back to the index view of the orders page and see all your other orders.  Those that have completed payment types should NOT have a 'cancel order' affordance next to them.

# Checklist:
